data/cataicon.svg

```diff
@@ -0,0 +1,257 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Is the image in [data/xdg/cataclysm-dda.svg](https://github.com/CleverRaven/Cataclysm-DDA/tree/master/data/xdg/cataclysm-dda.svg) not good? Why do we need two icon files?

Reply:
Definitely missed that. Let me modify and commit.